_episodes/07-aliases.md

```diff
@@ -77,3 +60,63 @@ $ git hi
 - How many times should you wait before aliasing a command?
 - Do you believe a list of generic two-letter acronyms for common commands will
   save your time?
+
+---
+
+## List of aliases the instructors use
+
+You are welcome to reuse, suggest, improve.
+
+Common aliases:
+
+```
+$ git config --global alias.br branch
+$ git config --global alias.ci "commit -v"
+$ git config --global alias.co checkout
+$ git config --global alias.di diff
+$ git config --global alias.di diff --cached --color-words
+$ git config --global alias.diw "diff --color-words"
+$ git config --global alias.dis "!git --no-pager diff --stat"
+$ git config --global alias.fe fetch
+$ git config --global alias.graph "log --all --graph --decorate --oneline"
+$ git config --global alias.rem remote
+$ git config --global alias.st status
+```
+
+Some common `-p` aliases:
```

Comment:
I'd propose the `-p` and submodule aliases be added to the main list... I left them separate because   After you merge I'll go through and describe what each command is used for (in a separate bulleted list below it).

Reply:
OK, I will fix that